codewiki/src/config.py

```
@@ -177,13 +182,14 @@ def from_cli(
             max_token_per_leaf_module: Maximum tokens per leaf module
             max_depth: Maximum depth for hierarchical decomposition
             agent_instructions: Custom agent instructions dict
-            
+            respect_gitignore: Respect .gitignore patterns during analysis
+
         Returns:
             Config instance
         """
         repo_name = os.path.basename(os.path.normpath(repo_path))
```

Ruff (0.14.13)

Local variable `repo_name` is assigned to but never used

Remove assignment to unused variable `repo_name`

(F841)